Sources/SimultaneouslyScrollView/SimultaneouslyScrollViewHandlerFactory.swift

```diff
@@ -1,10 +1,20 @@
 import Foundation
 
 /// Factory class to create `SimultaneouslyScrollViewHandler` instance
+@available(iOS 13, *)
+@available(tvOS, unavailable)
+@available(macOS, unavailable)
 public class SimultaneouslyScrollViewHandlerFactory {
+#if os(iOS)
     /// Creates a new `SimultaneouslyScrollViewHandler` instance
     /// - Returns: A new `SimultaneouslyScrollViewHandler` instance
     public static func create() -> SimultaneouslyScrollViewHandler {
         DefaultSimultaneouslyScrollViewHandler()
     }
+#else
+    public static func create() -> SimultaneouslyScrollViewHandler {
+        // Stub for other platforms
+        fatalError()
+    }
```

Comment:
This is needed to satisfy the compiler **AND** not introduce new unhelpful error messages.